newsletter.txt

```diff
@@ -0,0 +1,8 @@
+Visit Website: The user goes to the website where they want to subscribe to the newsletter.
```

Comment:
Format all points with a serial number or turn them into bullet points for readability.